src/Concerns/TestCase.php

```diff
@@ -130,16 +135,16 @@ public function toString(): string
     /**
      * Runs the test.
      */
-    public function __test(): void
+    public function __test()
     {
-        $this->__callClosure($this->__test, func_get_args());
+        return $this->__callClosure($this->__test, func_get_args());
     }
 
-    private function __callClosure(Closure $closure, array $arguments): void
+    private function __callClosure(Closure $closure, array $arguments)
```

Comment:
The return type should be `mixed`. If there is phpstan issues I will fix them in my side later.